experimental/examples/bubblebench/ot/src/proxy/tree.ts

```diff
@@ -55,33 +55,36 @@ const createObjectProxy = (
     consumer: Consumer,
     parent?: IProxy,
     parentKey?: json1.Key,
-) => new Proxy(subject, {
-    get: (target, key, receiver) => {
-        if (key === contextSym) {
-            return { parent, parentKey };
-        }
+) => {
+    const handler: ProxyHandler<IProxy> = {
+        get: (target, key, receiver) => {
+            if (key === contextSym) {
+                return { parent, parentKey };
+            }
 
-        const value = target[key];
+            const value = target[key];
 
-        /* eslint-disable @typescript-eslint/no-unsafe-return */
-        return value !== null && typeof value === "object"
-            ? getProxy(/* target: */ value, consumer, /* parent: */ receiver, key as string)
-            : value;
-        /* eslint-enable @typescript-eslint/no-unsafe-return */
-    },
-    set: (target, key, value, receiver) => {
-        const path = getPath(receiver, key as json1.Key);
+            /* eslint-disable @typescript-eslint/no-unsafe-return */
+            return value !== null && typeof value === "object"
+                ? getProxy(/* target: */ value, consumer, /* parent: */ receiver, key as string)
+                : value;
+            /* eslint-enable @typescript-eslint/no-unsafe-return */
+        },
+        set: (target, key, value, receiver) => {
+            const path = getPath(receiver, key as json1.Key);
```

Comment:
The indent formatting in this file seems suspicious... But I'm looking at the GitHub diff so that might be throwing me off.

Reply:
This is intended. Typescript made changes to the typing for this object (ProxyHandler specifically) to be more strict. To infer correctly (and get much better type errors) it is being assigned to an explicitly typed const instead of inlined, increasing the indent by one.